### PR TITLE
Improve authentication documentation

### PR DESCRIPTION
**Pull request by ravage84**
Various improvements

- Formatting
- Wording
- Grammar error
- Code, class name & path highlighing
- Namings
- De-emphasize less importaant remarks
- Update broken links

### CODE REVIEW

**Comment by ravage84** [comment]
@TerryKern can you have a look at this, please?

File: Docs/Documentation/Authentication.md [hunk 1]
```diff
@@ -169,13 +168,14 @@ The default configuration are:
...
-If the configuration is not enough for your project you may create a custom loader extending the
-default provided.
+To make the integration with CakePHP Authenication plugin easier we load the authenticators and identifiers
+defined at the ``Auth`` configuration key.
 
-- Create file src/Loader/AppAuthenticationServiceLoader.php
+If the default configuration is not enough for your project's needs you may create a custom loader extending the
+default loader provided.
+
+Fore xample, create a file ``src/Loader/AppAuthenticationServiceLoader.php``:
```
Comment (rochamarcelo) [inline Docs/Documentation/Authentication.md]: Typo in Fore xample.

**Review by rochamarcelo — CHANGES_REQUESTED** [review]

**Review by TerryKern — APPROVED** [review]
Looks good to me

**Review by rochamarcelo — APPROVED** [review]

**Comment by rochamarcelo** [comment]
Thank you.

**Pull request merged**